exporters/geneva/test/metrics_exporter_test.cc

```diff
@@ -23,6 +23,10 @@ std::string kUnixDomainPathAbstractSocket = "@/tmp/ifx_unix_socket";
 const std::string kNamespaceName = "test_ns";
 const std::string kAccountName = "test_account";
 
+const std::string kCustomNamespaceName = "custom_ns";
+const std::string kCustomAccountName = "custom_account";
```

Comment:
Are these 2 constants used?

Reply:
Good catch. I missed to check-in the latest changes to enable tests for custom namespace and account. Have added them now.